### ISSUE TITLE
Event subscribe is broken when initCCP is re-execute

### ISSUE THREAD

**Issue by HAYASHI-Masayuki** [amazon-connect/amazon-connect-streams#809]
Version: v2.6.2 or later

If `initCCP` is executed once to initialize, then uninitialized by executing `terminate`, and then initialized again, the process equivalent to `setupInitCCP` in `initCCP` is not executed. This leads to problems such as not being able to receive events subsequently.

Below is the minimum code to reproduce the problem and the procedure to reproduce it.

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <script type="text/javascript" src="connect-streams-min.js"></script>
  </head>
  <!-- Add the call to init() as an onload so it will only run once the page is loaded -->
  <body onload="init()">
    <button onclick="openPopup()">Open Login Popup Manually</button>
    <div id="container-div" style="width: 400px; height: 800px;"></div>
    <script type="text/javascript">
      var containerDiv = document.getElementById("container-div");
      var instanceURL = "https://my-instance-domain.my.connect.aws/ccp-v2/";
      // initialize the streams api
      function init() {
        // initialize the ccp
        connect.core.initCCP(containerDiv, {
          ccpUrl: instanceURL,            // REQUIRED
          loginPopup: false,              // optional, defaults to `true`
        });

        window.open(instanceURL, window.connect.MasterTopics.LOGIN_POPUP);

        connect.core.getEventBus().subscribe(window.connect.AgentEvents.REFRESH, () => {
          console.log('REFRESH received');
        });
      }

      function openPopup() {
        connect.core.terminate();
        init();
      }
    </script>
  </body>
</html>
```

Steps to Reproduce:

1. Open the HTML while not logged in.
2. Log in through the popup login window.
3. REFRESH event is received. This continues subsequently.
4. Log out and then manually open the login window.
5. REFRESH event is not received.

This issue may be related to #806.

**Comment by roman-v1** [comment]
I am facing the same issue.

**Comment by andywang219** [comment]
Hello,

We are currently investigating this issue and will update this thread with more information as soon as possible

**Comment by tannera926** [comment]
What version of Streams are you using? This issue was fixed in v2.10.0: https://github.com/amazon-connect/amazon-connect-streams/releases/tag/v2.10.0. Let us know if upgrading the version fixes the issue.
